projects/plugins/jetpack/modules/protect/blocked-login-page.php

```diff
@@ -376,7 +493,7 @@ function display_page( $title, $message, $back_button = false, $recovery_form =
 				echo "<meta name='robots' content='noindex,nofollow' />\n";
 			}
 			?>
-			<title><?php echo $title ?></title>
+			<title><?php echo $title; //phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped -- we set the $title ourselves. ?></title>
```

Comment:
Given that we call `protect_die` and set different titles in different places, maybe we should escape anyway, to ensure no HTML tags come messing things up?